pkg/controllers/management/feature/suite_test.go

```diff
@@ -0,0 +1,105 @@
+package feature
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/lasso/pkg/controller"
+	"github.com/rancher/rancher/pkg/wrangler"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+)
+
+var (
+	ctx              context.Context
+	cancel           context.CancelFunc
+	testEnv          *envtest.Environment
+	cfg              *rest.Config
+	k8sClient        client.Client
+	wranglerContext  *wrangler.Context
+	sharedController controller.SharedController
+)
+
+func TestFeatures(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Feature Suite")
+}
+
+var _ = BeforeSuite(func() {
+	ctx, cancel = context.WithCancel(context.TODO())
+
+	var err error
+	testEnv = &envtest.Environment{
+		CRDDirectoryPaths:     []string{filepath.Join("..", "..", "..", "crds", "yaml", "generated", "management.cattle.io_features.yaml")},
+		ErrorIfCRDPathMissing: true,
+	}
+	cfg, err = testEnv.Start()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
CI in drone is failing here, maybe you have to install [setup-envtest](https://github.com/rancher/fleet/blob/master/.github/scripts/run-integration-tests.sh#L9-L12) ?